python/xorbits/_mars/services/task/task_info_collector.py

```diff
@@ -45,11 +48,26 @@ async def wrapper(self, *args, **kwargs):
 
 
 class TaskInfoCollector:
-    def __init__(self, address: str, collect_task_info: bool = False):
-        self.collect_task_info = collect_task_info
-        self.result_chunk_to_subtask = dict()
+    def __init__(
+        self,
+        session_id: str,
+        task_id: str,
+        address: str,
+        collect_task_info: Optional[bool] = None,
+    ):
+        self._session_id = session_id
+        self._task_id = task_id
         self._address = address
-        self._task_api = None
+        self._result_chunk_to_subtask = dict()
+        self._save_dir = os.path.join(self._session_id, self._task_id)
```

Comment:
Can we support a path prefix via configuration and a filesystem like s3?

If user does not config the path prefix,  should we use a unified prefix for managing these files conveniently?

Reply:
There's another issue for this feature: https://github.com/xprobe-inc/xorbits/issues/214

Will do it after this PR got merged.